src/fragments/lib/storage/js/download.mdx

```diff
@@ -32,13 +35,11 @@ The `download` option send you the object data for download or programmatic mani
 const result = await Storage.get(`filename.txt`, { download: true });
 
 // data.Body is a Blob
-result.Body.text().then(string => { 
-  // handle the String data return String 
+result.Body.text().then((string) => {
+  // handle the String data return String
 });
 ```
 
-Note that the `Blob` methods like `.text()` are not supported on [IE/Opera/Safari](https://developer.mozilla.org/en-US/docs/Web/API/Blob/text); in those cases you can [parse manually](https://developer.mozilla.org/en-US/docs/Web/API/Blob#JavaScript).
```

Comment:
Just to double check, this is being removed because it's not longer relevant for our supported browsers?

Reply:
Yes, Correct currently .text() method is being supported in the browsers.